workers/python/main.py

```diff
@@ -0,0 +1,31 @@
+import asyncio
+import signal
+
+from logger import logger
+from workers import run_workers, shutdown_workers
+from config import RUN_BULLMQ_WORKERS
+
+
+async def main():
+
+    # Create an event that will be triggered for shutdown
+    shutdown_event = asyncio.Event()
+
+    def signal_handler(signal, frame):
+        logger.info("Signal received, shutting down.")
+        shutdown_event.set()
+
+    # Assign signal handlers to SIGTERM and SIGINT
+    signal.signal(signal.SIGTERM, signal_handler)
+    signal.signal(signal.SIGINT, signal_handler)
+
+    await run_workers(RUN_BULLMQ_WORKERS)
+    logger.info("Workers started")
+
+    # Wait until the shutdown event is set
+    await shutdown_event.wait()
+    await shutdown_workers()
+    logger.info("Workers shut down successfully.")
+
+if __name__ == "__main__":
+    asyncio.run(main())
```

Comment:
We will need a healthcheck server.

(additionally, the run_workers and shutdown_workers and signals can be handled in the lifecycle method of the server)

Reply:
Assuming you'd want to use the healtcheck server for k8s liveness probe. What should be its liveness condition?